expressEngine/test/helper.js

```diff
@@ -1,14 +1,21 @@
-import { connect } from 'mongoose';
-import inMemoryDB from './db';
+import mongoose from 'mongoose';
+import { MongoMemoryServer } from 'mongodb-memory-server';
 
+let mongoServer;
 
-let db;
-
-before(async () => {
-  const { mongoUri } = await inMemoryDB();
-  db = await connect(mongoUri, { useCreateIndex: true, useNewUrlParser: true });
+beforeEach((done) => {
+  mongoServer = new MongoMemoryServer();
```

Comment:
Is there is any reason for not adding `const` before mongoServer, at the time of initialization?
`mongoServer = new MongoMemoryServer();` to `const mongoServer = new MongoMemoryServer();`